### DISCUSSION TITLE
Video resolution

### DISCUSSION THREAD

**Discussion by alexanderadam** [KyleKun/one_second_diary#88] [Q&A] [answered]
Hi @KyleKun,

how is the default resolution defined?
And is there any possibility to influence it?

**Comment by KyleKun** [comment, marked as answer]
Hi @alexanderadam, it's 1080p, applied by ffmpeg. Generally speaking, recording with native camera on a good device will provide better image quality than recording in-app, but there is currently no way to influence the final output resolution.
